frontend/src/components/pages/RegistrantExperience/RegistrationSteps/PersonalInfo/personalInfoReducer.tsx

```diff
@@ -19,6 +19,11 @@ export const CamperReducer = (
     const newCampers: RegistrantExperienceCamper[] = JSON.parse(
       JSON.stringify(campers),
     ); // Deep Copy
+
```

Comment:
Do we also need this for the additionalInfoReducer?

Reply:
From my testing no we don't, since the other reducer doesn't use JSON.parse/stringify in this same way.